Cargo.toml

```diff
@@ -10,10 +10,13 @@ readme = "README.md"
 [features]
 vendored = ["openssl/vendored"]
 
+[dependencies]
+lazy_static = "1.0"
+rustc-serialize = "0.3"
```

Comment:
Why rustc-serialize?

Reply:
I think this is now no longer necessary?